src/sections/Blog/paginate.js

```diff
@@ -0,0 +1,90 @@
+import React,{useState} from "react";
+import { Link } from "gatsby";
+import { IoIosArrowRoundForward } from "@react-icons/all-files/io/IoIosArrowRoundForward";
```

Comment:
@subhradwip01 could you please use MUI icons only. Are these react-icons used anywhere else also?

Reply:
These react icons are also used in another pagination component...From there I take the reference of that icon... If you want I will change it

Reply:
Gotcha, thanks for confirming :+1: